verticapy/performance/vertica/qprof_utility.py

```diff
@@ -113,6 +113,32 @@ def _get_metrics() -> list:
             "mem_all_mb",
         ]
 
+    @staticmethod
+    def _get_metrics_name(metric: str, inv: bool = False) -> str:
+        look_up_table = {
+            "cost": "Query plan cost",
+            "rows": "Estimated row count",
+            "exec_time_ms": "Execution time in ms",
+            "est_rows": "Estimated row count",
+            "proc_rows": "Processed row count",
+            "prod_rows": "Produced row count",
+            "rle_prod_rows": "Produced RLE row count",
+            "clock_time_us": "Clock time in us",
```

Comment:
maybe change it to microseconds? instead of us?
or use "\u00B5"

Reply:
don't think it is necessary otherwise we will have to do it for everything